fem/datacollection.cpp

```diff
@@ -977,6 +978,7 @@ void ParaViewDataCollection::Save()
          pvtu_out << "<PDataArray type=\"" << GetDataTypeString()
                   << "\" Name=\"" << q_field_name
                   << "\" NumberOfComponents=\"" << vec_dim << "\" "
+                  << VTKComponentLabels(vec_dim) << " "
```

Comment:
So I could see this leading to issues down the road for certain users if they do depend on how ParaView automatically adds the labels to the components for them. 

This can probably be a follow-up PR, but it might make sense to add a high level function that let's the user/app developer specifically specify the labels for each component. Then down here if a user did specify those names those would be used here but if not it would revert to the component labels scheme you've added in this PR.